pkg/router/router_test.go

```diff
@@ -408,6 +418,110 @@ func TestConfigTemplate(t *testing.T) {
 				},
 			},
 		},
+		"Simple global HTTP request header": {
+			mustCreateWithConfig{
```

Comment:
Are these run?  Why don't they have the `mustCreate` stanza like the other tests?

Reply:
This will be set by ingress controller so setting field in route is not required